Add /events/ section

vkrasnovyd opened the pull request:
1. Set up the /events/ section with the following structure:
content/events/
├── 2024/
│   ├── workshop.md
│   ├── community-days.md
│   └── ...
└── 2025/
    └── ...

2. For /events/ created a template that separately renders upcoming and past events based on the flag in the /content/events/_index.md file.
3. Set the styling for the pages in the /events/ section and created helper shortcodes.
4. Migrated workshop 2024 and community days 2024 to this site.
5. Created archetypes (pre-filled templates) for the pages in the /events/ section and updated README file.

tschmidtb51 reviewed (changes requested):
Hmm... I think there was a misunderstanding: The events page should just link to the events - the events should be in their corresponding folder. Please see [FIRST's](https://www.first.org/events/) structure:
So it would be:
```
/events/
/workshop/
|-2024
|-2025
/community-days/
|-2024
|-2025
```

Please also provide more details on the event page (see FIRST as example).

bernhardreiter reviewed (dismissed):
LGTM & works.

bernhardreiter commented:
@tschmidtb51 this is ready to be merged. (Merge #11 first as I've merged that branch in here.)

vkrasnovyd commented:
@bernhardreiter 
> The instructions for creating event entries should also mention where other media files go, like pdf or odp presentation or video files that shall be offered for download. Ideally there should be one location for one event.

As discussed, I recommend placing downloadable files for each event in a separate folder named `download`. I've updated the README with instructions and examples of the folder structure for these files. I've also added a `download` folder to the 2024 directories for both event types as an example.

> @vkrasnovyd am I right in that those files could all go to `/static/images/events/workshops/2025/`? Would it make the structure more intuitive if we leave out `/images` from the path completely? (Check back with me before implementing something.)

You're right - having an `images` folder becomes less intuitive if we start adding other types of files. I've moved the nested folders from `static/images/` directly into `static/`, and updated all references and docs accordingly.

tschmidtb51 reviewed (dismissed):
I'll merge this for now. Please see necessary content fixes #13 and preparations for the merge to OASIS #12.

tschmidtb51 reviewed (dismissed):
Let's try again

tschmidtb51 reviewed (approved):
Go again
